Makefile

```diff
@@ -27,9 +27,31 @@ GO_VERSION_KIALI = 1.8.3
 
 # Identifies the docker image that will be built and deployed.
 DOCKER_NAME ?= kiali/kiali
-DOCKER_VERSION ?= dev
+DOCKER_VERSION ?= latest
```

Comment:
do we want to default to "latest" here?

Devs are the ones normally working with the Makefile, so I assume we want to keep this at "dev". CIs will normally be passing in the DOCKER_VERSION they want, thus they will override this default.

Reply:
ah... sorry